.github/workflows/release.yml

```diff
@@ -210,7 +210,7 @@ jobs:
           SERVICE_NAME: ${{ steps.setup.outputs.service_name }}
         run: |
           if [ "$SERVICE_NAME" == "scim" ]; then
-            $DOCKER_CONTENT_TRUST==0 
+            DOCKER_CONTENT_TRUST==0
```

Comment:
Did the `export` here not work?

Reply:
```suggestion
            export DOCKER_CONTENT_TRUST=0
```

Reply:
I believe before it was `export $DOCKER_CONTENT_TRUST=0`. I messed the `$` during review

Reply:
Yes, it is updated now

Reply:
@Eebru-gzy It looks like it is still missing the `export`

Reply:
Added now